scripts/agent.yaml.tmpl

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sm-agent
+  labels:
+    app: sm-agent
+    env: development
+spec:
+  containers:
+    - image: grafana/synthetic-monitoring-agent:latest
+      name: agent
+      imagePullPolicy: Never
+      args:
+        - -api-server-address
+        - sm-api:4031
```

Comment:
This bit here is correct if you have access to a local SM API server, but most users won't.

So it makes sense to make that a parameter, like the API_TOKEN, and maybe default to `sm-api:4031` if it's not provided.

Reply:
Done. Added as a defaulted config.